mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSArea.kt

```diff
@@ -310,9 +310,12 @@ data class MPSArea(val repository: SRepository) : IArea, IAreaReference {
             NodeReference(projectRef)
         }
 
+        val resolvedNodeForProject = resolveNode(projectRef) ?: return null
+        val resolvedProject = (resolvedNodeForProject as MPSProjectAsNode).project
```

Comment:
Should we use a safe cast here and throw a more explicit error in case the node cannot be cast?